packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### New features and enhancements
 
+- Added functionality for pagination in job spool files and added the command's key binding
```

Comment:
Please add either an issue link or a link to this PR.

Also, please add some more details to the PR description itself 😋